package.json

```diff
@@ -13,14 +13,19 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "config": "^3.3.2",
```

Comment:
I don't know if this is a good idea. How are we going to deploy this on Heroku or as at Docker image somewhere?
In those places we can add env vars so there is no need to commit the .env file, but how are we going to configure these secret properties without this uncommited conf file? Maybe we should look again into your problem with the .env files instead of trying a different approach

Reply:
Sure, i didn't know how the env variables were being used in Heroku, only reason i changed to config was so that i could store the firebase service account object in the default.json file. I'll look at a workaround.

Reply:
i have removed this and moved it all back to use .env. tested and it works for firebase 👍